javascript/javascript3/week2/fetchandwait.js

```diff
@@ -0,0 +1,26 @@
+const astronautApi = "http://api.open-notify.org/astros.json";
+    const promise = new Promise((resolve, reject) => {
```

Comment:
you should use a better variable name than 'promise' => what does this promise represent?